docs/csharp/language-reference/keywords/using.md

```diff
@@ -11,13 +11,7 @@ ms.assetid: 124e1a63-2a4b-4132-b269-3b6d8d3ef72d
 ---
 # using (C# Reference)
 
-The `using` keyword has two major uses:
-
-- As a directive, when it is used to create an alias for a namespace or to import types defined in other namespaces. See [using directive](using-directive.md).
-
-- As a statement, when it defines a scope at the end of which an object will be disposed. See [using statement](using-statement.md).
-
-In addition, the [using static](using-static.md) directive lets you define a type whose static members you can access without specifying a type name.
+The `using` keyword has two major uses. The [using statement](using-statement.md) defines a scope at the end of which an object will be disposed. The using [directive](using-directive.md) creates an alias for a namespace or imports types defined in other namespaces. The related [using static directive](using-static.md) imports the members of a single class.
```

Comment:
This is really 3 major uses.
I think that the original formatting is preferable, though -- at least, it makes the list-like content stand out more clearly.